PaloAltoNetworks Device Security (SCM) Content Pack (#44866)
demisto/content: Packs/PaloAltoNetworks_DeviceSecurity/Playbooks/playbook-device-security-check-servicenow_README.md

> This playbook should be used in a recurring Job to check the ServiceNow ticket status for the Palo Alto Networks Device Security (previously Zingbox) alerts or vulnerabilties.

Reviewer: typo: "vulnerabilties" ->"vulnerabilities"

Author: fixed typo: "vulnerabilties" ->"vulnerabilities"

Reviewer 2: ```suggestion
This playbook checks the ServiceNow ticket status for Palo Alto Networks Device Security (previously Zingbox) alerts or vulnerabilities. Designed to run as a recurring job.
```

Author: yes, i have re updated the description
